bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -131,6 +131,26 @@ public class FormContainerImpl extends AbstractContainerImpl implements FormCont
     @Default(values = DEFAULT_FORMS_SPEC_VERSION)
     private String specVersion;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @JsonIgnore
+    private String actionType;
+
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @JsonIgnore
+    private String[] mailto;
```

Comment:
Wouldn't mailTo etc anyways come, since today these are exposed in the default json as well. We should ideally add this to the blacklist from the model json

Reply:
Sure @rismehta , will make these properties part of reserved properties

Reply:
Resolving this as we decided to have the fd:submit namespace in Submission view and only when a certain header is passed.